docs/user-guide/kubectl/kubectl_convert.md

```diff
@@ -0,0 +1,124 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/user-guide/kubectl/kubectl_convert.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+## kubectl convert
+
+Convert config files between different API versions
+
+### Synopsis
+
+
+Convert config files between different API versions. Both YAML
+and JSON formats are accepted.
+
+The command takes filename, directory, or URL as input, and convert it into format
+of version specified by -output-version flag. If target version is not specified or
+not supported, convert to latest version.
+
+The default output will be printed to stdout in YAML format. One can use -o option
+to change to output destination.
+
+
+```
+kubectl convert -f FILENAME
+```
+
+### Examples
+
+```
+// Convert 'pod.yaml' to latest version and print to stdout.
+$ kubectl convert -f pod.yaml
+
+// Convert 'pod.yaml' to version v1 and output to 'b3pod.yaml'.
+$ kubectl convert -f pod.yaml --output-version v1 -t b3pod.yaml
```

Comment:
I don't see this implemented. Also, please do not use -t.

https://github.com/kubernetes/kubernetes/blob/master/docs/devel/kubectl-conventions.md#flag-conventions